vtr_flow/tasks/mixing_optimization/correctness_custom/mults_and_adders_simple/config/config.txt

```diff
@@ -0,0 +1,21 @@
+##############################################
```

Comment:
move this file in odin benchmarks. maybe make a vtr_flow directory, since this is not part of the regular flow, we dont want to pollute